packages/management/tanzu-auth/bundle/values.yaml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+
+tanzuAuth:
+  namespace: tanzu-auth
+image: "" #! the image to use for the tanzu-auth-controller-manager, there is no default image provided
```

Comment:
nit: seems weird that this isn't under `tanzuAuth`

Reply:
Yeah I agree.  I kind of dislike just calling it `image` as well.  There is a `controller` image and a `package` image, this is an image within the package, so its logically the controller image, but it could be clearer?

Reply:
Removed `tanzuAuth`. Its unecessary.